tests/components/stream/test_hls.py

```diff
@@ -1,20 +1,77 @@
 """The tests for hls streams."""
 from datetime import timedelta
+import io
 from unittest.mock import patch
 from urllib.parse import urlparse
 
 import av
+import pytest
 
 from homeassistant.components.stream import create_stream
+from homeassistant.components.stream.const import MAX_SEGMENTS, NUM_PLAYLIST_SEGMENTS
+from homeassistant.components.stream.core import Segment
 from homeassistant.const import HTTP_NOT_FOUND
 from homeassistant.setup import async_setup_component
 import homeassistant.util.dt as dt_util
 
 from tests.common import async_fire_time_changed
 from tests.components.stream.common import generate_h264_video
 
-
-async def test_hls_stream(hass, hass_client, stream_worker_sync):
+STREAM_SOURCE = "some-stream_source"
+SEQUENCE_BYTES = io.BytesIO(b"some-bytes")
+DURATION = 1000
```

Comment:
```suggestion
DURATION = 10
```
Doesn't really matter but 1000 seems quite large